installer/manifests/keptn/charts/control-plane/templates/core.yaml

```
@@ -205,6 +205,8 @@ spec:
               value: {{ .Values.bridge.automaticProvisioningMsg }}
             - name: RESOURCE_SERVICE_ENABLED
               value: {{.Values.resourceService.enabled | quote}}
+            - name: D3_HEATMAP_ENABLED
+              value: {{.Values.bridge.d3heatmap.enabled | quote}}
```

please add the desired default value here as well. this will ensure that upgrades go smoothly even if the `d3heatmap` value is not defined in the `values.yml` file of a user.
```suggestion
              value: {{ .Values.bridge.d3heatmap.enabled | default false | quote }}
```